README.md

```diff
@@ -1,9 +1,31 @@
-[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)
+<div align="center">
+
+<img src="https://raw.githubusercontent.com/quran/quran_android/master/app/src/madani/res/drawable-xxhdpi/icon.png" alt='Quran for Android logo'/>
+
 # Quran for Android
-[<img align="right" alt="Get it on Google Play" height="128" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png">](https://play.google.com/store/apps/details?id=com.quran.labs.androidquran)
+
+[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)
```

Comment:
not using travis anymore, using GitHub for the build now since it closed down - is there a way we can switch this to there?

Reply:
How about this:
[![Build Status](https://github.com/quran/quran_android/actions/workflows/build.yml/badge.svg)](https://github.com/quran/quran_android/actions/workflows/build.yml)

Reply:
looks good jazakAllah khair

Reply:
should i update the name on https://github.com/quran/quran_android/blob/master/.github/workflows/build.yml to `build`?